.github/workflows/os-llvm-sycl-build.yml

```diff
@@ -0,0 +1,77 @@
+name: Build with Open Source LLVM-Sycl compiler
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  install-compiler:
+    name: Install Nightly build of DPC++ toolchain
+    runs-on: ubuntu-20.04
+
+    env:
+      OCLCPUEXP_FN: oclcpuexp-2021.12.6.0.19_rel.tar.gz
+      FPGAEMU_FN: fpgaemu-2021.12.6.0.19_rel.tar.gz
+      TBB_FN: oneapi-tbb-2021.4.0-lin.tgz
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Download nightly and components
+        shell: bash -l {0}
+        run: |
+          cd /home/runner/work
+          mkdir -p sycl_bundle
+          cd sycl_bundle
+          git clone http://github.com/intel/llvm.git llvm_src
+          cd llvm_src
+          alias urlencode='python3 -c "import sys, urllib.parse as ul; print (ul.quote_plus(sys.argv[1]))"'
+          export NIGHTLY_TAG=$(urlencode $(git tag | sort -V | tail -1))
+          cd ..
+          wget https://github.com/intel/llvm/releases/download/${NIGHTLY_TAG}/dpcpp-compiler.tar.gz
+          wget https://github.com/intel/llvm/releases/download/2021-07/${OCLCPUEXP_FN}
+          wget https://github.com/intel/llvm/releases/download/2021-07/${FPGAEMU_FN}
```

Comment:
copy-pasta?

Reply:
URL prefixes are the same, I pushed a change to use env variable to store this prefix.